app/views/catalog/_export_button.html.erb

```diff
@@ -20,5 +20,9 @@
         <%= link_to content_tag('i', '', :class => 'icon-file-text-o') + t('ddf.tools.ris'), catalog_path(document, :format => 'ris') %>
       </li>
     <% end %>
+
+    <li>
+      <%= link_to t('ddf.tools.email_result'), citations_new_path(id: document.id), data: { ajax_modal: 'trigger'} %>
```

Comment:
Only resource urls i think. As regards commit squashing, I don't see the
benefit of losing that history. What's the argument for doing so?

man. 9. nov. 2015 16.40 skrev flyingzumwalt <notifications@github.com>:

> In app/views/catalog/_export_button.html.erb
> <https://github.com/dtulibrary/ddf/pull/2#discussion_r44288613>:
>
> > @@ -20,5 +20,9 @@
> >          <%= link_to content_tag('i', '', :class => 'icon-file-text-o') + t('ddf.tools.ris'), catalog_path(document, :format => 'ris') %>
> >        </li>
> >      <% end %>
> > +
> > +    <li>
> > +      <%= link_to t('ddf.tools.email_result'), citations_new_path(id: document.id), data: { ajax_modal: 'trigger'} %>
>
> Go with whatever rails is doing out of the box. I just thought that it set
> up named routes as edit_resource_path, new_resource_path, etc.
> … <#msg-f:1517377135009622256_>
> On Mon, Nov 9, 2015 at 1:50 PM, Ronan McHugh <notifications@github.com>
> wrote: In app/views/catalog/_export_button.html.erb <#2 (comment)
> <https://github.com/dtulibrary/ddf/pull/2#discussion_r44270925>>: > @@
> -20,5 +20,9 @@ > <%= link_to content_tag('i', '', :class =>
> 'icon-file-text-o') + t('ddf.tools.ris'), catalog_path(document, :format =>
> 'ris') %> > </li> > <% end %> > + > + <li> > + <%= link_to
> t('ddf.tools.email_result'), citations_new_path(id: document.id), data: {
> ajax_modal: 'trigger'} %> probably would be better alright, 'citations_new`
> was the helper generated by Rails from my route, but I can manually
> configure something more appropriate. — Reply to this email directly or
> view it on GitHub <
> https://github.com/dtulibrary/ddf/pull/2/files#r44270925>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dtulibrary/ddf/pull/2/files#r44288613>.
>